refinery/templates/data_set_manager/import.html

```diff
@@ -18,7 +18,7 @@
 <link
   rel="stylesheet"
   type="text/css"
-  href="{% static "vendor/intro.js/minified/introjs.min.css" %}">
```

Comment:
That's what release testing is for :) Actually, `GET vendor/intro.js/minified/introjs.min.css` was giving a 404, so this change can only be for the better :)